AssetBundles/conversation.manifest

```diff
@@ -40,5 +30,4 @@ ClassTypes:
 SerializeReferenceClassIdentifiers: []
 Assets:
 - Assets/DialogueBubble.prefab
-- Assets/GameAssets/Texture2D/UI_DialogueBox_d.png
```

Comment:
?

Reply:
yeah i have no clue. unity just did that on it's own